deploy/kubernetes/auditbeat-kubernetes.yaml

```diff
@@ -150,6 +172,10 @@ spec:
         hostPath:
           path: /var/lib/auditbeat-data
           type: DirectoryOrCreate
+      - name: run-containerd
+        hostPath:
+          path: /run/containerd
+          type: DirectoryOrCreate
```

Comment:
what happens here if you are not using containerd?

Reply:
An empty directory is created. I considered commenting out the code and explaining that it has to be commented in if file_integrity is used with the files of the containers, but I think this is easier this way for the user and not so bad in general.
When using docker as runtime this directory is also created, but it doesn't contain the root filesystems.
With cri-o this directory doesn't exist, so with this config it is created empty.

Reply:
Works for me, I wonder if all this deserves some mention in https://www.elastic.co/guide/en/beats/auditbeat/master/running-on-kubernetes.html

Reply:
Added a note about this in the docs.